paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsViewModel.kt

```diff
@@ -178,6 +183,12 @@ internal class PaymentOptionsViewModel @Inject constructor(
         clearErrorMessages()
 
         selection.value?.let { paymentSelection ->
+            if (paymentSelection is PaymentSelection.Link) {
+                val linkState = args.state.paymentMethodMetadata.linkState
+                val prominenceEnabled = linkState !== null && linkProminenceFeatureProvider
+                    .show2FADialogOnLinkSelectedInFlowController(linkState)
+                if (prominenceEnabled) { logger.debug("should show Link 2FA") }
+            }
```

Comment:
Let's remove this, since it's just used for logging.

Reply:
https://github.com/stripe/stripe-android/pull/10806/commits/ce75582d78b84db70450cf5169500aedc02c5df0